PhysicsTools/NanoAOD/plugins/SimpleFlatTableProducerPlugins.cc

```diff
@@ -9,11 +9,21 @@ typedef EventSingletonSimpleFlatTableProducer<GenEventInfoProduct> SimpleGenEven
 #include "SimDataFormats/HTXS/interface/HiggsTemplateCrossSections.h"
 typedef EventSingletonSimpleFlatTableProducer<HTXS::HiggsClassification> SimpleHTXSFlatTableProducer;
 
+
+#include "DataFormats/ProtonReco/interface/ForwardProton.h"
+typedef SimpleFlatTableProducer<reco::ForwardProton> SimpleProtonTrackFlatTableProducer;
+
+#include "DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h"
+typedef SimpleFlatTableProducer<CTPPSLocalTrackLite> SimpleLocalTrackFlatTableProducer;
+
 #include "DataFormats/Math/interface/Point3D.h"
 typedef EventSingletonSimpleFlatTableProducer<math::XYZPointF> SimpleXYZPointFlatTableProducer;
 
 #include "FWCore/Framework/interface/MakerMacros.h"
 DEFINE_FWK_MODULE(SimpleCandidateFlatTableProducer);
 DEFINE_FWK_MODULE(SimpleGenEventFlatTableProducer);
 DEFINE_FWK_MODULE(SimpleHTXSFlatTableProducer);
+DEFINE_FWK_MODULE(SimpleProtonTrackFlatTableProducer);
+DEFINE_FWK_MODULE(SimpleLocalTrackFlatTableProducer);
 DEFINE_FWK_MODULE(SimpleXYZPointFlatTableProducer);
```

Comment:
remove DEFINE_FWK_MODULE(SimpleXYZPointFlatTableProducer); 
it was added below

Reply:
Done.